searchcore/src/vespa/searchcore/proton/server/fast_access_doc_subdb.cpp

```diff
@@ -275,6 +273,9 @@ FastAccessDocSubDB::applyConfig(const DocumentDBConfig &newConfigSnapshot, const
             reconfigureAttributeMetrics(*newMgr, *oldMgr);
         }
         _iFeedView.set(_fastAccessFeedView.get());
+        if (isNodeRetired()) {
+            reconfigureAttributesConsideringNodeState();
```

Comment:
Attribute vectors on retired node will get a config glitch that is fixed up here, but might have spent some time performing compaction due to the compaction strategy temporarily being set as if the node was not retired.

Reply:
Then the modified config will be persisted and un retirement will not work.